gcampus/core/models/index/structure.py

```diff
@@ -0,0 +1,452 @@
+#  Copyright (C) 2021-2022 desklab gUG (haftungsbeschränkt)
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Affero General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Affero General Public License for more details.
+#
+#  You should have received a copy of the GNU Affero General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from __future__ import annotations
+
+__ALL__ = ["StructureIndex"]
+
+from typing import Union
+
+from django.contrib.gis.db import models
+from django.utils.translation import gettext_lazy as _
+
+from gcampus.core.models.index.base import WaterQualityIndex
+
+
+class UtilizationCategory(models.TextChoices):
+    NATURAL = "natural", _("natural forest (deciduous trees)")
+    EXTENSIVE = "extensive", _(
+        "extensive use: unfertilized or small grazed meadows, no buildings"
+    )
+    MIXED = "mixed", _("smaller fields, pastures or gardens, coniferous forest")
+    INTENSIVE = "intensive", _("intensive farming and arable land, some buildings")
+    ARTIFICIAL = "artificial", _("village, industrial area")
+    NA = "unknown", _("unknown")
```

Comment:
There is also `__empty__` which corresponds to an empty (`NULL` or `""`) value. Using a designated field like you did could be helpful as we differentiate between **unknown** and **not specified, or empty**. Should we keep it like this?

Reply:
Yes, this was intended to be used when the parameter has been measured but it could not be categorised, so i would prefer to keep it like this and use the empty value if it has not been measured.